client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -2,20 +2,80 @@
  * External dependencies
  */
 import * as React from 'react';
+import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies.
  */
-import TotalPaymentsVolume from './total-payments-volume';
-import PaymentsDataHighlights from './payments-data-highlights';
+import PaymentsDataTile from './payments-data-tile';
+import {
+	TotalPaymentsVolumeTooltip,
+	PaymentsDataChargeTooltip,
+	PaymentsDataFeesTooltip,
+} from './payments-data-highlights-tooltips';
 
 import './style.scss';
+import { getAdminUrl } from 'wcpay/utils';
 
 const PaymentsActivityData: React.FC = () => {
 	return (
-		<div className="payments-activity-data">
-			<TotalPaymentsVolume />
-			<PaymentsDataHighlights />
+		<div className="wcpay-payments-activity-data">
+			<PaymentsDataTile
+				id="wcpay-payments-activity-data__total-payments-volume"
+				label={ __( 'Total payments volume', 'woocommerce-payments' ) }
+				currencyCode="EUR"
+				amount={ 156373 }
+				tooltip={ <TotalPaymentsVolumeTooltip /> }
```

Comment:
⛏️ Since these tooltips are only used here, we could add them inline here to make the code slightly easier to follow (arguably). I'm happy either way, though.

```suggestion
				tooltip={
					<ClickTooltip
						className="total-payments-volume__tooltip"
						buttonIcon={ <HelpOutlineIcon /> }
						buttonLabel={ __(
							'Total payments volume tooltip',
							'woocommerce-payments'
						) }
						content={ __(
							'test total payments volume content',
							'woocommerce-payments'
						) }
					/>
				}
```

Reply:
I like the idea of keeping things flat where possible. In this case, though, I am leaning towards keeping the tooltips in a separate file since, I am expecting more content within each of them. It may be cleaner to keep it this way.

Reply:
I'd prefer to keep the markup flat, use `ToolTip` component inline. As @Jinksi suggests, these tooltips are only used once, so the abstraction just makes it harder to iterate on the UI in future.

Reply:
Ah. Thanks, @haszari . The point about the Tooltip being used only once is a very valid one to tilt this in favor of flattening this. I will create a follow-up refactor PR to fix this. 👍

Reply:
I have placed a refactor PR here :
[Refactor: Flatten out tooltips in Payments activity widget UI. #8619](https://github.com/Automattic/woocommerce-payments/pull/8619)